task/prepare.go

```diff
@@ -0,0 +1,80 @@
+package task
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/golang/glog"
+	livepeerAPI "github.com/livepeer/go-api-client"
+	"github.com/livepeer/go-livepeer/drivers"
+	"github.com/livepeer/stream-tester/model"
+	"github.com/livepeer/stream-tester/segmenter"
+)
+
+func RecordStream(ctx context.Context, lapi *livepeerAPI.Client, playbackID string, osSess drivers.OSSession) (string, error) {
+	glog.Infof("made it into record function")
+	fullPath := videoFileName(playbackID)
+	glog.Infof("full path %s", fullPath)
+	fir, err := osSess.ReadData(ctx, fullPath)
```

Comment:
Would be cool to make this function receive an `io.Reader` instead and just run it in parallel with the regular `Probe` and `Save` tasks in the `Import` function. Not a must though.

Reply:
I would like to do this, though for transcode it would need to run in sequence, so will leave this for later to reduce complexity for now

Reply:
Yeah certainly not a must!